### ISSUE TITLE
extract: Plural-Forms header should be fixed up

### ISSUE THREAD

**Issue by mikix** [edx/i18n-tools#68]
$ i18n_tool extract
$ grep Plural-Forms conf/locale/en/LC_MESSAGES/django-partial.po
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

Which then blows up python-gettext with errors like:

ValueError: invalid token in plural form: EXPRESSION

I think it's reasonable for extract.py to -- along with its other header cleanup it does -- also clean up that header to look like the normal en locale setting:

"Plural-Forms: nplurals=2; plural=(n != 1);\n"

**Comment by l1ph0x** [comment]
Hi Mikix,
I am running a native install of open-release/hawthorn.master. Today I have pulled a new language translations from tx using paver i18n_robot_pull. This resulted in the .mo .po files for djangojs-studio, django-studio, djangojs-partial and django-partial for default language (EN) all having "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n" I had to manually redo them as mentioned in your note. I still cannot compile assets even if i18n_tool validate does not show any errors anymore. Now the failure to build must be due to a different / unrelated reason.
